### ISSUE TITLE
Return xpath-builder object

### ISSUE THREAD

**Issue by Airborn22** [featurist/css-to-xpath#2]
Hi @joshski,

First of all I want to thank you for the great job, I really like your module :)
The only thing I miss is that currently I can't get the xpath-builer object when selecting an element. There are things which are not selectable with css, for example the content of a node. It would be great if you would return only `parse(selector)` on line 15 of `css_to_xpath.js` so you can continue to build your xpath object and then call `toXPath()` only when you need it. Something like this:
```javascript
var cssToXPath = require('css-to-xpath');
var x = require('xpath-builder').dsl();
cssToXPath('.modal-dialog .modal-title').where(x.text().equals('something')).toXPath()
```

I was thinking to make a pull request for that but I don't know if it would brake compatibility for others or if there is a nicer solution but if you say, I can do that :)

It would be also nice to be able get the reference of the xpath-builder module from your module so you don't have to load it while using css-to-xpath. The example above would be then something like
```javascript
var cssToXPath = require('css-to-xpath');
cssToXPath('.modal-dialog .modal-title').where(cssToXPath.x.text().equals('something')).toXPath()
```

Thank you,
Alex

**Comment by joshski** [comment]
Hi @Airborn22,

Glad you like it!

This all sounds like it makes a lot of sense. I'm not aware of any other users of this module, so if you want to make a pull request then I will merge it...

Cheers

**Comment by Airborn22** [comment]
Hi, @joshski,

Thank you! Finally I noticed that you already export the parse() method so there is no need to break the backward compatibility. I just exported the xpath-builder to be able to use it from outside. Pull request created: https://github.com/featurist/css-to-xpath/pull/3/


Thank you,
Alex

**Comment by joshski** [comment]
Fixed in 0.1.0.

Thanks Alex!
